### PR TITLE
feat(rhineng-12779): Pass HBI schema using update to cyndi config map

### PR DESCRIPTION
**Pull request by chambridge**
* Add default value of public
* Add deployment parameter specification to add HBI schema to config map
* Update controller logic to gather HBI schema value from config and construct inventory table name
* Add basic test cases

### CODE REVIEW

**Review by jpramos123 — APPROVED** [review]

File: controllers/config/defaults.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package config
 const defaultTopic = "platform.inventory.events"
 const defaultConnectCluster = "xjoin-kafka-connect-strimzi"
```
Comment (thearifismail) [inline controllers/config/defaults.go:4]: Should move to using `insights-kafka-connect` because that's what Devprod has moved to (per @SteveHNH ) because `xjoin` is not used anymore.  This frees us up from the maintenance of `xjoin-kafka-connect-strimzi` image.  May be this should be done in a different PR

**Review by thearifismail — APPROVED** [review]
this LGTM but please look at the comment about using `insights-kafka-connect`, which is now used by the devprod team.

File: controllers/config/defaults.go (hunk 1 above)
Comment (kruai) [reply controllers/config/defaults.go:4]: Yes, I think that's a change for a different PR. It would probably make more sense to be done as part of the xjoin-kafka-connect-strimzi vulnerabilities Jira

**Review by kruai — APPROVED** [review]
